app/react-client-app/src/components/EditTable.jsx

```diff
@@ -1,130 +1,175 @@
     return (
-        <div>
-            <CCBreadcrumbBar>
-                <Link href={"/"}>Home</Link>
-                <Link href={"/scanreports/"}>Scan Reports</Link>
-                <Link href={`/scanreports/${scanReport.current.id}/`}>{scanReport.current.dataset}</Link>
-                <Link href={`/scanreports/${scanReport.current.id}/tables/${table.id}/`}>{table.name}</Link>
-                <Link href={`/scanreports/${scanReport.current.id}/tables/${table.id}/update/`}>Update</Link>
-            </CCBreadcrumbBar>
-            <PageHeading text={"Update Table"} />
-            {isOpen &&
-                <ScaleFade initialScale={0.9} in={isOpen}>
-                    <ToastAlert hide={onClose} title={alert.title} status={alert.status} description={alert.description} />
-                </ScaleFade>
-            }
-            <CCSelectInput
-                id={"table-person-id"}
-                label={"Person ID"}
-                selectOptions={fields.map((item) => item.name)}
-                handleInput={setPerson}
-                isDisabled={!canEdit}
-            />
-            <CCSelectInput
-                id={"table-date-event"}
-                label={"Date Event"}
-                selectOptions={fields.map((item) => item.name)}
-                handleInput={setDate}
-                isDisabled={!canEdit}
-            />
-            {canEdit &&
-                <Button
-                    bgColor="#3db28c"
-                    mt="10px"
-                    onClick={updateTable}
-                >
-                    Update {table.name}
-                </Button>
-            }
-        </div>
+      <div>
+        <Flex padding="30px">
+          <Spinner />
+          <Flex marginLeft="10px">
+            {loadingMessage ? loadingMessage : "Loading page"}
+          </Flex>
+        </Flex>
+      </div>
     );
-}
+  }
+  return (
+    <div>
+      <CCBreadcrumbBar>
+        <Link href={"/"}>Home</Link>
+        <Link href={"/scanreports/"}>Scan Reports</Link>
+        <Link href={`/scanreports/${scanReport.current.id}/`}>
+          {scanReport.current.dataset}
+        </Link>
+        <Link
+          href={`/scanreports/${scanReport.current.id}/tables/${table.id}/`}
+        >
+          {table.name}
+        </Link>
+        <Link
+          href={`/scanreports/${scanReport.current.id}/tables/${table.id}/update/`}
+        >
+          Update
+        </Link>
+      </CCBreadcrumbBar>
+      <PageHeading text={"Update Table"} />
+      {isOpen && (
+        <ScaleFade initialScale={0.9} in={isOpen}>
+          <ToastAlert
+            hide={onClose}
+            title={alert.title}
+            status={alert.status}
+            description={alert.description}
+          />
+        </ScaleFade>
+      )}
+      <Flex paddingTop={5} paddingBottom={5}>
+        <Alert status="info">
+          <AlertIcon />
+          Mapping Rules cannot be generated without the Person ID and Date Event
```

Comment:
I think this could be made clearer. Perhaps 
```
Mapping Rules cannot be generated without the Person ID and Date Event being set.
Once you set these, Mapping Rules will be generated for all Concepts currently associated to the table.
```

Reply:
Have applied this change